cmd/metrics-server/app/start.go

```diff
@@ -49,6 +49,12 @@ func runCommand(o *options.Options, stopCh <-chan struct{}) error {
 		fmt.Println(version.VersionInfo())
 		os.Exit(0)
 	}
+
+	errors := o.Validate()
+	if len(errors) > 0 {
+		return errors[0]
```

Comment:
we need to raport all errors we got from validate.

I think we should
* fmt.Printf errors one by one
* return new validation error